src/coreclr/debug/ee/loongarch64/dbghelpers.S

```diff
@@ -19,7 +19,7 @@ NESTED_ENTRY FuncEvalHijack, _TEXT, UnhandledExceptionHandlerUnix
     // push arg to the stack so our personality routine can find it
     // push lr to get good stacktrace in debugger
```

Comment:
Could you help to modify the `lr` as `ra` ?
```suggestion
    // push ra to get good stacktrace in debugger
```

Reply:
Also the Line11:
```
// a0 : pointer to DebuggerEval object
```

Thanks

Reply:
```diff
diff --git a/src/coreclr/debug/ee/loongarch64/dbghelpers.S b/src/coreclr/debug/ee/loongarch64/dbghelpers.S
index 07103d6c677..25452aa6a5d 100644
--- a/src/coreclr/debug/ee/loongarch64/dbghelpers.S
+++ b/src/coreclr/debug/ee/loongarch64/dbghelpers.S
@@ -8,7 +8,7 @@
 // hijacking stub used to perform a func-eval, see Debugger::FuncEvalSetup() for use.
 //
 // on entry:
-//   x0  : pointer to DebuggerEval object
+//   a0  : pointer to DebuggerEval object
 //
 
 // @dbgtodo- once we port Funceval, use the ExceptionHijack stub instead of this func-eval stub.
@@ -17,9 +17,9 @@ NESTED_ENTRY FuncEvalHijack, _TEXT, UnhandledExceptionHandlerUnix
     //       you change the prolog you will also need to update the personality routine.
 
     // push arg to the stack so our personality routine can find it
-    // push lr to get good stacktrace in debugger
+    // push ra to get good stacktrace in debugger
     //                            $fp,$ra
-    PROLOG_SAVE_REG_PAIR_INDEXED  22, 1, -32
+    PROLOG_SAVE_REG_PAIR_INDEXED  22, 1, 32
     st.d  $a0, $sp, 16
 
     // FuncEvalHijackWorker returns the address we should jump to.
diff --git a/src/coreclr/debug/ee/riscv64/dbghelpers.S b/src/coreclr/debug/ee/riscv64/dbghelpers.S
index 71b584d7aa6..9376760b8fb 100644
--- a/src/coreclr/debug/ee/riscv64/dbghelpers.S
+++ b/src/coreclr/debug/ee/riscv64/dbghelpers.S
@@ -17,8 +17,8 @@ NESTED_ENTRY FuncEvalHijack, _TEXT, UnhandledExceptionHandlerUnix
     //       you change the prolog you will also need to update the personality routine.
 
     // push arg to the stack so our personality routine can find it
-    // push lr to get good stacktrace in debugger
-    PROLOG_SAVE_REG_PAIR_INDEXED  fp, ra, -32
+    // push ra to get good stacktrace in debugger
+    PROLOG_SAVE_REG_PAIR_INDEXED  fp, ra, 32
     sd  a0, 16(sp)
 
     // FuncEvalHijackWorker returns the address we should jump to.
diff --git a/src/coreclr/pal/src/arch/loongarch64/activationhandlerwrapper.S b/src/coreclr/pal/src/arch/loongarch64/activationhandlerwrapper.S
index 9cc8663fedf..59ad00dd293 100644
--- a/src/coreclr/pal/src/arch/loongarch64/activationhandlerwrapper.S
+++ b/src/coreclr/pal/src/arch/loongarch64/activationhandlerwrapper.S
@@ -11,7 +11,7 @@ C_FUNC(ActivationHandlerReturnOffset):
 
 NESTED_ENTRY ActivationHandlerWrapper, _TEXT, NoHandler
     //                           $fp,$ra
-    PROLOG_SAVE_REG_PAIR_INDEXED 22, 1, -(16 + CONTEXT_Size)
+    PROLOG_SAVE_REG_PAIR_INDEXED 22, 1, (16 + CONTEXT_Size)
     // Should never actually run
     EMIT_BREAKPOINT
     bl      EXTERNAL_C_FUNC(ActivationHandler)
diff --git a/src/coreclr/pal/src/arch/loongarch64/dispatchexceptionwrapper.S b/src/coreclr/pal/src/arch/loongarch64/dispatchexceptionwrapper.S
index 0548f03462f..d29913c377d 100644
--- a/src/coreclr/pal/src/arch/loongarch64/dispatchexceptionwrapper.S
+++ b/src/coreclr/pal/src/arch/loongarch64/dispatchexceptionwrapper.S
@@ -34,7 +34,7 @@ C_FUNC(PAL_DispatchExceptionReturnOffset):
 
 NESTED_ENTRY PAL_DispatchExceptionWrapper, _TEXT, NoHandler
     //                           $fp,$ra
-    PROLOG_SAVE_REG_PAIR_INDEXED 22, 1, -16
+    PROLOG_SAVE_REG_PAIR_INDEXED 22, 1, 16
     // Should never actually run
     EMIT_BREAKPOINT
     bl      EXTERNAL_C_FUNC(PAL_DispatchException)
```

Reply:
@t-mustafin 
Could you help to udate all these together ?
Thanks very much.

Reply:
Thank you!